source/modules.tex

```diff
@@ -49,7 +49,7 @@
 
 \pnum
 A \defn{module interface unit} is a module unit whose
-\grammarterm{module-declaration} contains the \tcode{export} keyword;
+\grammarterm{module-declaration} contains an \grammarterm{export-keyword};
```

Comment:
This should say "whose /module-declaration/ starts with /export-keyword/." for a bit more precision. "contains" is usually danger-zone wording (it might transitively contain). Except with the special case of /export-keyword/, we're probably safe.